eclair-core/pom.xml

```diff
@@ -34,7 +34,7 @@
             <plugin>
                 <groupId>com.googlecode.maven-download-plugin</groupId>
                 <artifactId>download-maven-plugin</artifactId>
-                <version>1.3.0</version>
+                <version>1.7.0</version>
```

Comment:
On my dev machine (ubuntu), bitcoin core is not cached anymore but is downloaded every time I run tests, it seems to be linked to https://github.com/maven-download-plugin/maven-download-plugin/issues/246

Reply:
Good catch, I'm indeed reproing this! I'll downgrade to 1.6.8

Reply:
Downgraded in https://github.com/ACINQ/eclair/pull/2710/commits/62ccecb2db0a9bdb27239d562e217ae974e1ed1c, I tested that this version correctly loads binaries from the cache.